src/main/java/org/lmdbjava/Env.java

```diff
@@ -226,6 +231,77 @@ public boolean isReadOnly() {
   }
 
   /**
+   * Provides the name of all {@link Dbi}s currently stored in this
+   * {@code Env}.
+   *
+   * <p>
+   * Note, this method, creates its own read-only transaction and
+   * can not be nested inside other transactions in some cases.
+   *
+   * @return A {@code List} containing the name of every {@code Dbi}
+   *   in this {@code Env}.  (never null)
+   */
+  public List<String> allDbiNames() {
+    if (closed) {
+      throw new AlreadyClosedException();
+    }
+    final List<String> result = new ArrayList<>();
+    try (Txn<byte[]> txn
+        = new Txn<>(this, null, ByteArrayProxy.PROXY_BA, MDB_RDONLY_TXN)) {
+      final Dbi<?> names = new Dbi<>(this, txn, null);
+      try (Cursor<byte[]> cursor = names.openCursor(txn)) {
```

Comment:
with 'openCursor' no longer copuled to the Dbi buffer type, I can just use the `byte[]` variant here to trivially get the String's bytes.